format-common/src/main/java/io/cdap/plugin/format/plugin/AbstractFileSourceConfig.java

```diff
@@ -210,4 +237,16 @@ public Schema getSchema() {
   public boolean shouldCopyHeader() {
     return copyHeader;
   }
+
+  /**
+   * Takes the first word of the file encoding string, as any further words are just charset descriptions.
+   *
+   * @param fileEncoding The file encoding parameter supplied by the user
+   * @return the cleaned up file encoding name
+   */
+  public static String cleanFileEncodingName(String fileEncoding) {
```

Comment:
ideally we would just configure the UI to use a different display name than the actual value instead of performing some cleanup in the backend. However, I don't think there's a way to do this for the select widget.  @elfenheart can you confirm?

Reply:
Looks like there is a way, using:

```
options: [ { label: 'this is the label', value: 'value' }  ]
```

Reply:
I didn't know the UI supported this configuration format. I'll fix it.